app/templates/rest_framework/base.html

```diff
@@ -1,19 +1,58 @@
 {% extends "rest_framework/base.html" %}
 {% load static %}
-
 {% block style %}
 {{ block.super }}
-<link rel="icon" href="{% static 'favicon.ico' %}">
+    <link rel="icon" href="{% static 'style/favicon.ico' %}">
+    <link rel="stylesheet" href="{% static 'style/style.css' %}" type="text/css">
+{% endblock %}
+{% block title %}Geoadmin STAC API{% endblock %}
```

Comment:
Is this block title required ? if yes I would also use the same title as in line 15.

Reply:
The default comportment of the title block is to get the title of wherever we are ( as in the breadcrumbs menu. For example, /api/stac/v0.9/collections/ch.swisstopo.swisstlmregio/items would be "Item List") as the page <title>, followed by Django REST framework. using data.geo.admin.ch STAC API is cleaner. So I changed it to that.